test/testRunner.js

```diff
@@ -317,35 +346,273 @@ describe('JES explorer function verification tests', () => {
+                });
+
+                // raised github issue #84 against explorer-jes repo, remove skip after that resolution
+                it.skip('Should handle a none recognised query parameter gracefully (still load page)', async () => {
+                    const expectedFilter = {
+                        owner: 'IZU*',
+                        prefix: 'pre*',
+                        jobId: 'st',
+                        status: 'INPUT',
+                    };
+                    const filters = { ...DEFAULT_SEARCH_FILTERS, ...expectedFilter, ...{ noParam: 'noValue' } };
+
+                    await loadUrlWithSearchFilters(driver, filters);
+
+                    expect(await testFilterDisplayStringValues(driver, expectedFilter)).to.be.true;
+                    expect(await testFilterFormInputValues(driver, expectedFilter)).to.be.true;
+                });
+            });
+            describe('Owner Url Filter', () => {
+                it('Should handle setting owner from url query', async () => {
+                    const filters = { owner: 'IZUSVR' };
+                    const expectedFilter = { ...DEFAULT_SEARCH_FILTERS, ...filters };
+                    const expectedJobs = ['IZU', 'ZOWE', 'ZWE'];
+
+                    await loadUrlWithSearchFilters(driver, filters);
+
+                    expect(await testFilterDisplayStringValues(driver, expectedFilter)).to.be.true;
+                    expect(await testJobOwnerFilter(driver, expectedJobs)).to.be.true;
+                });
+            });
+            describe('Prefix Url Filter', () => {
+                it('Should handle fetching jobs based on full prefix (ZOWE_JOB_NAME)', async () => {
+                    const filters = { prefix: ZOWE_JOB_NAME };
+                    const expectedFilter = { ...DEFAULT_SEARCH_FILTERS, ...filters };
+                    const expectedPrefix = ZOWE_JOB_NAME;
+
+                    await loadUrlWithSearchFilters(driver, filters);
+
+                    expect(await testFilterDisplayStringValues(driver, expectedFilter)).to.be.true;
+                    expect(await testJobPrefixFilter(driver, expectedPrefix)).to.be.true;
+                });
+
+                it('Should handle fetching jobs based on prefix with asterisk (ZOWE*)', async () => {
+                    const expectedPrefix = 'ZOWE*';
+                    const filters = { prefix: expectedPrefix };
+                    const expectedFilter = { ...DEFAULT_SEARCH_FILTERS, ...filters };
+
+                    await loadUrlWithSearchFilters(driver, filters);
+
+                    expect(await testFilterDisplayStringValues(driver, expectedFilter)).to.be.true;
+                    expect(await testJobPrefixFilter(driver, expectedPrefix)).to.be.true;
+                });
+                // TODO::rework test once we have message in the tree showing no jobs found
+                it('Should handle fetching no jobs based on crazy prefix (1ZZZZZZ1)', async () => {
```

Comment:
In Zowe 1.0.1 we return a single job instance with text 'No jobs found' so the expect needs to be changed to accommodate this

Reply:
this needs to be tested once, added changes related to zowe 1.0.1